automatic/dropbox/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$packageArgs        = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = $evn:ChocolateyPackageTitle
```

Comment:
please set the software name manually, don't rely on `ChocolateyPackageTitle` as this was first added in 0.10.1